### PR TITLE
Signup Ui

### LINKED ISSUE #4: Signup UI
### Signin UI need to be done!

**Without some parts:**
<del>Signup title at the top of the window</del>
<del>Close icon at the top left of the window</del>
<del>Repeat password input</del>

*Change location of `Agree to our Terms & Stuff` to be under the Signup button, without checkbox input and change message to be `By clicking Signup, you agree with our Terms & Privacy`*

Join the Discord channel for this issue [Signup Channel](https://discord.gg/46mB95)

Compete with colleagues in #3  
Let's see result better than them!

**Don't forget to check [Atlas Track project](https://github.com/PresKhaled/atlas/projects/1) instructions (Information, Pinned columns) before contribution**

**UI**

Signup UI

### PR DESCRIPTION
**Pull request by gor3a**
* Not Done:
1- "Terms & Stuff Page" no design for this 

device-2020-04-25-193232

### Second commit edit status bar 
device-2020-04-25-200710

### CODE REVIEW

**Review by PresKhaled — APPROVED** [review]
Excellent work! ✨

**Comment by PresKhaled** [comment]
Good Job 💯 

There was a **conflicts** in "include.dart, sun.png".
  - always make sure you have the latest version of release-long-live before PR. [Term 2](https://github.com/PresKhaled/atlas/projects/1#card-36751503)

For "Terms & Privacy", it's a link, you can use [url_launcher package](https://pub.dev/packages/url_launcher) for this.

File: lib/account/signup.dart [hunk 1]
```diff
@@ -1,14 +1,376 @@
...
+  }
+
   @override
   Widget build(BuildContext context) {
-    return Scaffold(body: Container(child: Column(children: <Widget>[])));
+    SystemChrome.setSystemUIOverlayStyle(SystemUiOverlayStyle(
+        statusBarColor: Colors.transparent,
+        statusBarBrightness: Brightness.dark,
+        systemNavigationBarColor: Color.fromRGBO(193, 55, 111, 1),
+        systemNavigationBarIconBrightness: Brightness.light));
```
Comment (PresKhaled) [inline lib/account/signup.dart:37]: Make a method in include.dart

Comment (gor3a) [reply lib/account/signup.dart:37]: > Good Job 
> 
> There was a **conflicts** in "include.dart, sun.png".
> 
> * always make sure you have the latest version of release-long-live before PR. [Term 2](https://github.com/PresKhaled/atlas/projects/1#card-36751503)
> 
> For "Terms & Privacy", it's a link, you can use [url_launcher package](https://pub.dev/packages/url_launcher) for this.

I was take latest version , but I don't know what's the problem

Comment (gor3a) [reply lib/account/signup.dart:37]: > Make a method in include.dart

sure , and will create confirmation modal

**Pull request merged**